ql/src/test/results/clientpositive/llap/catalog.q.out

```diff
@@ -0,0 +1,119 @@
+#### A masked pattern was here ####
+PREHOOK: query: SHOW CATALOGS
+PREHOOK: type: SHOWCATALOGS
+POSTHOOK: query: SHOW CATALOGS
+POSTHOOK: type: SHOWCATALOGS
+hive
+test_cat
+PREHOOK: query: DROP CATALOG test_cat
+PREHOOK: type: DROPCATALOG
+PREHOOK: Input: catalog:test_cat
+PREHOOK: Output: catalog:test_cat
+POSTHOOK: query: DROP CATALOG test_cat
+POSTHOOK: type: DROPCATALOG
+POSTHOOK: Input: catalog:test_cat
+POSTHOOK: Output: catalog:test_cat
+PREHOOK: query: SHOW CATALOGS
+PREHOOK: type: SHOWCATALOGS
+POSTHOOK: query: SHOW CATALOGS
+POSTHOOK: type: SHOWCATALOGS
+hive
+#### A masked pattern was here ####
+PREHOOK: type: CREATECATALOG
+PREHOOK: Output: catalog:test_cat
+#### A masked pattern was here ####
+POSTHOOK: type: CREATECATALOG
+POSTHOOK: Output: catalog:test_cat
+#### A masked pattern was here ####
+PREHOOK: query: SHOW CATALOGS
+PREHOOK: type: SHOWCATALOGS
+POSTHOOK: query: SHOW CATALOGS
+POSTHOOK: type: SHOWCATALOGS
+hive
+test_cat
+PREHOOK: query: DROP CATALOG IF EXISTS test_cat
+PREHOOK: type: DROPCATALOG
+PREHOOK: Input: catalog:test_cat
+PREHOOK: Output: catalog:test_cat
+POSTHOOK: query: DROP CATALOG IF EXISTS test_cat
+POSTHOOK: type: DROPCATALOG
+POSTHOOK: Input: catalog:test_cat
+POSTHOOK: Output: catalog:test_cat
+PREHOOK: query: SHOW CATALOGS
+PREHOOK: type: SHOWCATALOGS
+POSTHOOK: query: SHOW CATALOGS
+POSTHOOK: type: SHOWCATALOGS
+hive
+PREHOOK: query: DROP CATALOG IF EXISTS test_cat
+PREHOOK: type: DROPCATALOG
+POSTHOOK: query: DROP CATALOG IF EXISTS test_cat
+POSTHOOK: type: DROPCATALOG
+#### A masked pattern was here ####
+PREHOOK: type: CREATECATALOG
+PREHOOK: Output: catalog:test_cat
+#### A masked pattern was here ####
+POSTHOOK: type: CREATECATALOG
+POSTHOOK: Output: catalog:test_cat
+#### A masked pattern was here ####
+PREHOOK: query: SHOW CATALOGS
+PREHOOK: type: SHOWCATALOGS
+POSTHOOK: query: SHOW CATALOGS
+POSTHOOK: type: SHOWCATALOGS
+hive
+test_cat
+PREHOOK: query: SHOW CATALOGS LIKE 'test%'
+PREHOOK: type: SHOWCATALOGS
+POSTHOOK: query: SHOW CATALOGS LIKE 'test%'
+POSTHOOK: type: SHOWCATALOGS
+test_cat
+PREHOOK: query: SHOW CATALOGS LIKE 'test_'
+PREHOOK: type: SHOWCATALOGS
+POSTHOOK: query: SHOW CATALOGS LIKE 'test_'
+POSTHOOK: type: SHOWCATALOGS
+PREHOOK: query: SHOW CATALOGS LIKE 'test__'
+PREHOOK: type: SHOWCATALOGS
+POSTHOOK: query: SHOW CATALOGS LIKE 'test__'
+POSTHOOK: type: SHOWCATALOGS
+#### A masked pattern was here ####
+PREHOOK: type: ALTERCATALOG_LOCATION
+PREHOOK: Output: catalog:test_cat
+#### A masked pattern was here ####
+POSTHOOK: type: ALTERCATALOG_LOCATION
+POSTHOOK: Output: catalog:test_cat
+#### A masked pattern was here ####
+PREHOOK: query: DESC CATALOG EXTENDED test_cat
+PREHOOK: type: DESCCATALOG
+PREHOOK: Input: catalog:test_cat
+POSTHOOK: query: DESC CATALOG EXTENDED test_cat
+POSTHOOK: type: DESCCATALOG
+POSTHOOK: Input: catalog:test_cat
+Catalog Name	test_cat	 	 
+Comment	Hive test catalog	 	 
+#### A masked pattern was here ####
+Create Time	2025-03-06 17:08:12.000	 	 
```

Comment:
`Create Time ` should be masked.

https://ci.hive.apache.org/blue/organizations/jenkins/hive-precommit/detail/PR-3288/6/tests
```
Error
Client Execution succeeded but contained differences (error code = 1) after executing catalog.q 
119c119
< Create Time 2025-03-06 20:17:53.000    
---
> Create Time 2025-03-06 17:08:12.000
```